test/setup/test_provisioner.py

```diff
@@ -20,13 +20,14 @@
 import os
 import traceback
 import unittest
+import yaml
 from cortx.provisioner.provisioner import CortxProvisioner
+from cortx.utils.conf_store import MappedConf, Conf
```

Comment:
This can be removed if not used in this code.

Reply:
done